src/Copy/BrandCopy.elm

```diff
@@ -464,7 +464,7 @@ brandCopy key =
             CopyText "After Hazel’s husband Colin was diagnosed with Progressive Supranuclear Palsy they both struggled to adjust and needed a little extra support."
 
         StoryTwoTeaserImageAlt ->
-            CopyText "\u{2728}"
+            CopyText "✨"
```

Comment:
Would you mind also adding some alt text for all the images that don't have it? Looks like maybe is the teaser imaged.

Reply:
Fixed it :)